app/views/words/_form_search.html.slim

```diff
@@ -1,5 +1,6 @@
 .search-form
   .input-group
-    = search_form_for(@search, url: search_path) do |f|
-      = f.search_field :ja_or_en_or_vi_cont, class: "form-search", placeholder: t(".your_search"), required: true, autocomplete: :off
-      = f.submit t(".search"), class: "btn btn-primary"
+    = form_with url: search_path, local: true, method: :get, html: { class: "form-inline my-2 my-lg-0" } do |f|
```

Comment:
Sao không dùng search_form_for nữa?

Reply:
E với anh Hùng thống nhất lại với nhau dùng form_with ạh

Reply:
Tai sao ko dung thong nhat la search_form_for

Reply:
Anh Hung dang lam search autocomplete, anh day dung search_form_for khong duoc nen em sua lai theo form_with anh day dang lam ah :))

Reply:
thế  thì Phải báo Hùng là tìm hiểu xem vì sao không được chứ. Cứ ko đc là bỏ đi ah?

Reply:
Kĩa năng có hạn, sửa mãi mà nó ko chạy đc a ạ. :v

Reply:
Anh day bao tim hieu mat hon 1 ngay khong duoc nen sua thanh the ah

Reply:
OK